internal/actioninfo/actioninfo.go

```diff
@@ -1,9 +1,28 @@
 package actioninfo
 
+import (
+	"fmt"
+	"log"
+)
+
 type DataParser interface {
-	// TODO: добавить методы
+	Parse(string) error
+	ActionInfo() (string, error)
 }
 
 func Info(dataset []string, dp DataParser) {
-	// TODO: реализовать функцию
+	for _, v := range dataset {
+		err := dp.Parse(v)
+		if err != nil {
+			log.Println("ошибка парсинга")
```

Comment:
Ошибку надо выводить, а то не понятно, что случилось
```suggestion
			log.Println("ошибка парсинга", err)
```